package.json

```diff
@@ -35,6 +35,7 @@
     "ember-data": "^2.7.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
+    "ember-factory-for-polyfill": "1.1.1",
```

Comment:
Could we move this to `dependencies` just to be sure this will work for apps running ember < 2.3?